configure.ac

```diff
@@ -546,7 +546,28 @@ esac
 
 AC_ARG_WITH(iodbc,
 AS_HELP_STRING([--with-iodbc=DIR], [build odbc driver against iODBC in DIR]))
-if test "x$with_iodbc" != "x" -a "x$with_iodbc" != "xno"; then
+
+if test "x$with_iodbc" = "xyes"; then
+	# The user asked for iodbc support, but didn't tell us
+	# where to look. So, we'll try to get the info from
+	# pkg-config. If we can't, we'll display an error.
+	PKG_CHECK_MODULES(IODBC, libiodbc)
+	CPPFLAGS="$CPPFLAGS -DIODBC $IODBC_CFLAGS"
```

Comment:
Not a problem adding to global but I would add IODBC_CFLAGS to ODBC_INC for compatibility.